plugins/modules/cloud_account.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2024, Network to Code (@networktocode) <info@networktocode.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: cloud_account
+short_description: Creates or removes cloud_account from Nautobot
+description:
+  - Creates or removes cloud_account from Nautobot
+notes:
+  - Tags should be defined as a YAML list
+  - This should be ran with connection C(local) and hosts C(localhost)
+author:
+  - Travis Smith (@tsm1th)
+requirements:
+  - pynautobot
+version_added: "5.4.0"
+extends_documentation_fragment:
+  - networktocode.nautobot.fragments.base
+  - networktocode.nautobot.fragments.tags
+  - networktocode.nautobot.fragments.custom_fields
+options:
+  name:
+    description:
+      - The name of the cloud_account
+    required: true
+    type: str
+  account_number:
+    description:
+      - Required if I(state=present) and the cloud_account does not exist yet
+    required: false
+    type: str
+  description:
+    description:
+      - The description of the cloud_account
+    required: false
+    type: str
+  cloud_provider:
+    aliases:
+      - provider
+    description:
+      - Required if I(state=present) and the cloud_account does not exist yet
+    required: false
+    type: raw
+  secrets_group:
+    description:
+      - The secrets group of the cloud_account
+    required: false
+    type: raw
+"""
+
+EXAMPLES = r"""
+---
+- name: Create a cloud_account
+  networktocode.nautobot.cloud_account:
+    url: http://nautobot.local
+    token: thisIsMyToken
+    name: Cisco Quantum Account
+    cloud_provider: Cisco
```

Comment:
```suggestion
    provider: Cisco
```

Let's use the alias in our examples instead of the namespaced option we had to create to avoid collisions.

Reply:
You marked this as resolved, but it wasn't changed